tests/unit/test_ansible_inventory.py

```diff
@@ -84,6 +84,37 @@ def empty_layout():
     }
 
 
+@pytest.fixture()
+def metadata_extra():
+    hosts = [
+        {
+            "name": "srv1.example.test",
```

Comment:
Can we replace one data set without window os and with some other custom meta_* param and without meta_readonly_dc param? to just have different data to verify other non window and non meta_readonly_dc param data.

Reply:
Some of the uses cases with not having the values present are tested by another tests in the `test_ansible_inventory` output. 

Changed one of the hosts to not be a windows host.